files/default/neutron-ha-tool.py

```diff
@@ -646,6 +662,29 @@ def wait_router_ports_active(qclient, router_id, target_host, maxtries=60):
                            (router_id, ", ".join(remaining)))
 
 
+def ssh_delete_router_namespace(target_host, router_id):
+    namespace = "qrouter-" + router_id
+
+    ssh_client = paramiko.SSHClient()
+    ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
Not obvious what `AutoAddPolicy()` does here, so maybe you could add a comment explaining it?